src/app/components/home/home.component.html

```diff
@@ -0,0 +1,241 @@
+<header>
+  <div class="bg-light py-1">
+    <div class="container">
+      <div class="row">
+        <div class="col-md-6 col-12 text-center text-md-start">
+          <span> Super Value Deals - Save more with coupons</span>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="navbar navbar-light py-lg-4 pt-3 px-0 pb-0">
+    <div class="container">
+      <div class="row w-100 align-items-center g-lg-2 g-0">
+        <div class="col-xxl-2 col-lg-3">
+          <a class="navbar-brand d-none d-lg-block" routerLink="/">
+            <img
+              src="/assets/images/logo/freshcart-logo.svg"
+              alt="eCommerce HTML Template"
+            />
+          </a>
+          <div class="d-flex justify-content-between w-100 d-lg-none">
+            <a class="navbar-brand" routerLink="/">
+              <img
+                src="/assets/images/logo/freshcart-logo.svg"
+                alt="eCommerce HTML Template"
+              />
+            </a>
+            <div class="d-flex align-items-center lh-1">
+              <button
+                class="navbar-toggler collapsed"
+                type="button"
+                aria-controls="navbar-default"
+                aria-label="Toggle navigation"
+              >
+                <span class="icon-bar top-bar mt-0"></span>
+                <span class="icon-bar middle-bar"></span>
+                <span class="icon-bar bottom-bar"></span>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <ul class="nav pt-3">
+        <li *ngFor="let category of categories$ | async" class="nav-item">
+          <a class="nav-link" [routerLink]="category.link" role="button">
+            {{ category.name }}
+          </a>
+        </li>
+      </ul>
+    </div>
+  </div>
+</header>
+<div class="border-bottom pb-lg-0 pb-3 mb-lg-13">
+  <nav class="navbar navbar-expand-lg navbar-light navbar-default pt-0 pb-0">
+    <div class="container px-0 px-md-3">
+      <div class="offcanvas offcanvas-start p-4 p-lg-0">
+        <div
+          class="d-flex justify-content-between align-items-center mb-2 d-block d-lg-none"
+        >
+          <a routerLink="/"
+            ><img
+              src="/assets/images/logo/freshcart-logo.svg"
+              alt="eCommerce HTML Template"
+          /></a>
+          <button type="button" class="btn-close" aria-label="Close"></button>
+        </div>
+        <div class="d-block"></div>
+      </div>
+    </div>
+  </nav>
+</div>
+
+<footer class="footer">
+  <div class="container">
+    <div class="row g-4">
+      <div class="col-12 col-md-12 col-lg-4">
+        <div class="row g-4">
+          <div class="col-12">
+            <h6 class="mb-4">Categories</h6>
+            <ul class="navbar-nav pb-5">
+              <li *ngFor="let category of categories$ | async" class="nav-item">
```

Comment:
You subscribe to categories$ 2 times in this file. Do it only once and use it in both cases